portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmailReport.testcase

```diff
@@ -41,6 +41,68 @@ definition {
 		}
 	}
 
+	@description = "Story: LRAC-11664 | Automation ID: LRAC-11844 | Test Summary: Check that cancel button does not enable email report"
+	@priority = "3"
+	test CanCancelChangesToEmailReport {
+		task ("Sign in AC as Test Test user") {
+			ACUtils.launchAC();
+		}
+
+		task ("Go to property and Select the property you are using") {
+			ACNavigation.goToSettings();
+
+			ACSettings.goToProperties();
+
+			ACNavigation.openItem(itemName = "${assignedPropertyName}");
+		}
+
+		task ("Enable and configure email report frequency") {
+			ACEmailReport.accessEmailReportSettings();
```

Comment:
Hey, do you think the task could be rename to `Enable and configure email report frequency and then cancel the changes`? Just to be clearer, what do you think?